codalab/rest/bundles.py

```diff
@@ -426,6 +425,67 @@ def _fetch_locations():
     return dict(data=uuids_to_locations)
 
 
+@get('/bundle_stores/', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_stores():
+    """
+    Fetch the bundle stores available to the user.
+
+    Returns a dictionary in which the keys are the bundle store uuids, and the values
+    are tuples with the owner_id, name, and url.
+    """
+    return local.model.get_bundle_stores(request.user)
+
+
+@post('/bundle_stores/', apply=AuthenticatedProtectedPlugin())
+def _add_bundle_store():
+    """
+    Add a bundle store that the user can access.
+    """
+    return local.model.create_bundle_store(
+        request.user,
+        request.forms.get('name'), 
+        request.forms.get('storage_type'), 
+        request.forms.get('storage_format'), 
+        request.forms.get('url'), 
+        request.forms.get('authentication')
+    )
+
+
+@put('/bundle_stores/<id:re:%s>', apply=AuthenticatedProtectedPlugin())
+def _update_bundle_store(uuid):
+    """
+    Update a bundle store that the user can access.
+    """
+    return local.model.update_bundle_store(
+        request.user, 
+        uuid,
+        request.forms.get('name'), 
+        request.forms.get('storage_type'), 
+        request.forms.get('storage_format'), 
+        request.forms.get('url'), 
+        request.forms.get('authentication')
+    )
+
+
+@get('/bundle_stores/<id:re:%s>', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_store(uuid):
+    """
+    Fetch the bundle store corresponding to the specified id.
+
+    Returns a dictionary in which the key id the bundle store uuid, and the value
+    is a tuple with the owner_id, name, and url.
```

Comment:
Check `_fetch_bundle` to see how its output is returned, and return the output in a similar way.

Goal: we want all rest endpoints to return output in a predictable manner.